ingredient/ingredient-event-hub-namespace/src/functions.ts

```diff
@@ -2,11 +2,42 @@ import {BaseUtility, IngredientManager} from '@azbake/core'
 
 export class EventHubNamespaceUtils extends BaseUtility {
 
-    public create_resource_name(): string {
+    public get_resource_name(shortName: string | null = null): string {
         let util = IngredientManager.getIngredientFunction("coreutils", this.context)
 
-        const profile = util.create_resource_name("ehn", null, true);
-        return profile;
+        //resource type, name, region enabled
+        const fullName = util.create_resource_name("ehn", shortName, true);
+
+        this.context._logger.debug(`EventHubNamespaceUtils.get_resource_name() returned ${fullName}`);
+
+        return fullName;
     } 
+
+    public async get_resource_group(rgShortName: string | null = null): Promise<string> {        
```

Comment:
corutils has a resource_group function already. I think this can be removed?

Reply:
I believe I wanted to support getting the resource group of a resource deployed in a different recipe.  My other PR added support for that and I'll do the same here.